### ISSUE TITLE
Provide default value also for required fields

### ISSUE THREAD

**Issue by languitar** [koxudaxi/datamodel-code-generator#224]
**Is your feature request related to a problem? Please describe.**

Assume the following specifications:
```yaml
openapi: "3.0.0"
info:
  title: Sample API
  description: API description in Markdown.
  version: 1.0.0
servers: []
paths: {}
components:
  schemas:
    test:
      type: object
      properties:
        broken:
          type: string
          default: "a string"
          description: "dummy"
      required:
        - broken
```

Right now, the generated Field `broken` will be:
```python
broken: str = Field(..., description='dummy')
```

This is in line with the OpenAPI specification but especially for the purpose of generating data this is pretty annoying, because you always have to provide default values that are already known from the specification.

**Describe the solution you'd like**

In case of `required` + `default`, the `default` should still be spliced into the field. I don't see how this could harm apart from providing convenience.

**Describe alternatives you've considered**

If the fear is that error situations are missed when parsing inputs, an optional command line flag could be used to realize this.

**Additional context**

**Comment by koxudaxi** [comment]
@languitar 
Thank you for creating this issue.
Did you want an option like `--use-default` ?
This option sets default value on required field.

**Comment by languitar** [comment]
> Did you want an option like `--use-default` ?
> This option sets default value on required field.

If there's no chance to always set the defaults, that would be ok for me.

**Comment by koxudaxi** [comment]
@languitar 
I have released a new version as `0.5.35`
you can give `--use-default` to the command-line option.

**Comment by languitar** [comment]
I think this can be closed now. Thanks for implementing this.
